integrations/hotjar/lib/index.js

```diff
@@ -7,9 +7,7 @@ var is = require('is');
  * Expose `HotJar` integration
  */
 
-var Hotjar = (module.exports = integration('Hotjar')
-  .option('hjid', null)
-  .option('hjPlaceholderPolyfill', true));
```

Comment:
Why are we removing `hjPlaceholderPolyfill` option? :thinking:

Reply:
We no longer support this option.